source/common/decompressor/zlib_decompressor_impl.cc

```diff
@@ -68,7 +68,11 @@ bool ZlibDecompressorImpl::inflateNext() {
     return false; // This means that zlib needs more input, so stop here.
   }
 
-  RELEASE_ASSERT(result == Z_OK, "");
+  if (result < 0) {
+    decompression_error_ = result;
```

Comment:
suggest adding trace-log here with the error number & URL.

Reply:
@jmarantz to which URL are you referring?

Reply:
I added a URL to the definitions of the error codes.